src/events/processCollectedSignatures/index.js

```diff
@@ -92,11 +90,8 @@ function processCollectedSignaturesBuilder(config) {
               )
               return
             } else if (e instanceof IncompatibleContractError) {
-              logger.fatal(`The contract is not compatible: ${e.message}`)
-              process.exit(10)
-            } else if (e instanceof InvalidValidatorError) {
-              logger.fatal({ address: VALIDATOR_ADDRESS }, 'Invalid validator')
-              process.exit(10)
+              logger.error(`The contract is not compatible: ${e.message}`)
```

Comment:
It is not correct to say that "the contract is not compatible". The issue actually is with data collected by the contract.

Reply:
I grouped both problems, the number of signatures being less than required and the signatures being incorrect. In both cases, the message is ignored and the situations is logged.